samples/dependency-injection/externally-managed-mode/sample.md

```diff
@@ -1,33 +1,32 @@
 ---
-title: Externally Managed Container Mode Usage
-summary: A sample that uses the NServiceBus externally managed container mode to configure a DI container.
+title: Externally managed Container Mode Usage
+summary: A sample that uses NServiceBus externally managed container mode to configure a DI container.
 component: Core
-reviewed: 2020-09-04
+reviewed: 2020-09-17
 related:
  - nservicebus/dependency-injection
 ---
 
 ### Configuring an endpoint to use ServiceProvider
```

Comment:
ServiceProvider is mentioned in this title, but there is no reference to it anywhere in the code or text. It feels like something is missing to connect it with the content.